app/views/trends/_empty_data.html.erb

```diff
@@ -0,0 +1,7 @@
+<% title(_("Trends for %s") % trend_title(@trend)) %>
+<div class="row">
+  <div class="stats-well col-md-12">
+    <p><strong><%= _('No data for this trend.') %></strong></p>
+    <div><%= _("Is cron job that executes <span class='black'>foreman-rake trends:counter</span> enabled?").html_safe %></div>
```

Comment:
Since the command's not going to be translated, you can avoid having to translate HTML by doing something like: `(_("Is the cron job that executes %s enabled?") % "<span...</span>").html_safe`